proof/refine/AARCH64/orphanage/Orphanage.thy

```diff
@@ -915,20 +901,15 @@ proof -
     unfolding schedule_def
     apply (wp, wpc)
          \<comment> \<open>action = ResumeCurrentThread\<close>
-         apply (wp)[1]
+         apply wp
         \<comment> \<open>action = ChooseNewThread\<close>
         apply (clarsimp simp: when_def scheduleChooseNewThread_def)
         apply (wp ssa_no_orphans hoare_vcg_all_lift)
             apply (wp hoare_disjI1 chooseThread_nosch)
-           apply (wp nextDomain_invs_no_cicd' hoare_vcg_imp_lift
-                     hoare_lift_Pf2 [OF tcbQueued_all_queued_tcb_ptrs_lift
-                                      [OF nextDomain_tcbQueued]
-                                      nextDomain_ct']
-                     hoare_lift_Pf2 [OF st_tcb_at'_is_active_tcb_ptr_lift
-                                      [OF nextDomain_st_tcb_at']
-                                      nextDomain_ct']
-                     hoare_vcg_all_lift getDomainTime_wp)[2]
-          apply wpsimp
+           apply ((wpsimp wp: nextDomain_invs_no_cicd' hoare_vcg_imp_lift'
+                              st_tcb_at'_is_active_tcb_ptr_lift
+                              tcbQueued_all_queued_tcb_ptrs_lift hoare_vcg_all_lift getDomainTime_wp
+                   | wps)+)[2]
```

Comment:
very nice, that hoare_lift_Pf2 looks like something I used to do, glad to see it go